src/session_impl.cpp

```diff
@@ -5548,6 +5554,28 @@ namespace aux {
 			, boost::ref(m_alerts), _1, cb), salt);
 	}
 
+	void session_impl::dht_get_peers(sha1_hash const& info_hash)
+	{
+		if (!m_dht) return;
```

Comment:
Could you elaborate a little more about it? I made them as close as possible to dht_put_item and I'm not sure I understand.